backend/sozisel/lib/sozisel_web/schema/resolvers/participant_resolvers.ex

```diff
@@ -186,4 +191,90 @@ defmodule SoziselWeb.Schema.Resolvers.ParticipantResolvers do
 
     verify_launched_event(ctx, launched_event_id, on_verify)
   end
+
+  def insert_result_in_transaction(event_result_attrs, image, filename, extension) do
```

Comment:
You need to make this function more explicit as this file is named `participant_resolvers.ex` and this function does not suggest that you mean whiteboard.

You could event make the insert and run a separate functions that have `whiteboard` part in function name